src/gradient/sensitivity_calculator.py

```diff
@@ -1,27 +1,47 @@
+from src.static.cm_data_aggregate_kenya import KenyaDataAggregator
+
 from src.gradient.eigen_value_gradient import EigenValueGradient
+from src.gradient.ngm_gradient import NGMGradient
 from src.comp_graph.cm_creator import CMCreator
 from src.comp_graph.cm_elements_cg_leaf import CMElementsCGLeaf
-from src.gradient.ngm_gradient import NGMGradient
-# Import NGMCalculators from different models
+
+from src.static.cm_leaf_preparator import CGLeafPreparator
+from src.static.dataloader import DataLoader
+from src.static.eigen_calculator import EigenCalculator
+from src.static.model_base import EpidemicModelBase
+
 from src.models.chikina.ngm_calculator import NGMCalculator as ChikinaNGMCalculator
 from src.models.moghadas.ngm_calculator import NGMCalculator as MoghadasNGMCalculator
 from src.models.seir.ngm_calculator import NGMCalculator as SeirNGMCalculator
 from src.models.rost.ngm_calculator import NGMCalculator as RostNGMCalculator
 from src.models.italy.ngm_calculator import NGMCalculator as ItalyNGMCalculator
 from src.models.kenya.ngm_calculator import NGMCalculator as KenyaNGMCalculator
 from src.models.british_columbia.ngm_calculator import NGMCalculator as BCNGMCalculator
-# Other imports
-from src.static.cm_leaf_preparator import CGLeafPreparator
-from src.static.dataloader import DataLoader
-from src.static.eigen_calculator import EigenCalculator
+
+from src.models.rost.model import RostModelHungary
```

Comment:
Put all these imports into `__init__.py` located in `models` and here you can use only `import src.models as models` and refer to a model with `models`, e.g. `models.ChikinaModel` or `models.BCModel`